pkg/process/checks/net.go

```diff
@@ -208,6 +209,14 @@ func batchConnections(
 			EncodedDNS:        dnsEncoder.Encode(batchDNS),
 			ContainerHostType: cfg.ContainerHostType,
 		}
+
+		// Add OS telemetry
+		if hostInfo := host.GetStatusInformation(); hostInfo != nil {
+			cc.KernelVersion = hostInfo.KernelVersion
```

Comment:
Does this work on Windows?

Reply:
`KernelVersion` in particular will be empty, but the other params are filled here: https://github.com/DataDog/datadog-agent/blob/6d001423308b9d8ae5daf065ad9fec68a31bccfc/pkg/metadata/host/host_windows.go#L106-L132

Reply:
urgh, `KernelArch` is actually missing from the Windows struct. I can either add it and perhaps call `runtime.GOARCH` or get rid of it altogether. @leeavital thoughts?

Reply:
Should we be collecting something like `PlatformVersion` also so we have some versioning info for windows?

Reply:
@hmahmood I *think* `Platform` carries enough information. @stb2132 was kind enough to run an agent flare from a testing windows box:
```
  Host Info
  =========
    bootTime: 2020-11-05 19:10:40.000000 GMT
    os: windows
    platform: Windows Server 2019 Datacenter
    platformFamily: Windows Server 2019 Datacenter
    platformVersion: 10.0 Build 17763
    procs: 108
    uptime: 2h51m41s
```
Any thoughts?

Reply:
If it contains version information as well, then sure `Platform` should be ok as well. What is `Platform` on linux?

Reply:
ubuntu, debian etc